warehouse/jobs/handlers.go

```diff
@@ -18,36 +18,36 @@ import (
 )
 
 // AddWarehouseJobHandler The following handler gets called for adding async
-func (asyncWhJob *AsyncJobWhT) AddWarehouseJobHandler(w http.ResponseWriter, r *http.Request) {
-	pkgLogger.Info("[WH-Jobs] Got Async Job Add Request")
-	pkgLogger.LogRequest(r)
+func (a *AsyncJobWhT) AddWarehouseJobHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
`a` looks like too generic. rename to `asyncJob` ??

Reply:
[The name of a method's receiver should be a reflection of its identity; often a one or two letter abbreviation of its type suffices (such as "c" or "cl" for "Client"). ](https://github.com/golang/go/wiki/CodeReviewComments#receiver-names)